doc/contributing/technical-priorities.md

```diff
@@ -160,3 +160,18 @@ and integration with other systems.
 As TypeScript usage continues to grow and gains more prominence in the
 ecosystem, enhancing its support is essential for delivering an improved
 developer experience for newcomers and experienced users alike.
+
+## Non-goals
```

Comment:
Can we use some other word that "goal" please? Also it would help if it was phrased in a positive form (instead of "removing npm is not a priority", "npm users DX is a priority" or something like that)

Reply:
What would you propose? “npm users DX is a priority” is nowhere near specific enough to rule out unbundling npm. One could argue that Corepack managing npm is “improving npm users’ DX,” for example.

Reply:
Well, I'm suggesting to not rule out the unbundling of npm.

Reply:
No, that's the point of this PR 😄  If you want to block it, that's fine.

Reply:
Could you make the goal of the PR to document the consensus instead? That should lead to more productive discussions.

Reply:
That is the goal of this PR. I think I'm documenting the current consensus (or more precisely, the majority view).

We could have two PRs, one for each option, that are each blocked; but I'm not sure that's more efficient. If the TSC votes on this and the blocks aren't overturned, we can open the inverse thereafter and there shouldn't be any reason to block or vote on it because we'll have already settled the question.

Reply:
I think I’m going to mark as resolved because it’s not clear to me what else I can do to address your concern. My goal is not to document an amorphous goal like “improve DX for npm users.” My goal is to settle the question of whether or not we plan to remove npm in the near term.

If you’d like to suggest specific alternative text that aligns with my goal, please feel free to unresolve.

Reply:
One way I have seen this kind of thing phrased well is to use the "non-goal" as a "vs alternative" to clarify the intent. We use this at work in the form of "bets", but I think it would work well here called "goals". Something like:

```
# Goals

1. Providing a single package manager with stable DX is a priority (vs. providing many or none)
```

That way we are are still getting the point across and are clearly saying "not ship multiple" without having all the negativity.